components/errdoc/main.go

```diff
@@ -58,19 +58,53 @@ func searchError(args []string) error {
 		return err
 	}
 
-	result, err := index.Search(bleve.NewSearchRequest(bleve.NewMatchPhraseQuery(strings.Join(args, " "))))
+	// Bleve search
+	queries := []query.Query{
+		bleve.NewMatchPhraseQuery(strings.Join(args, " ")),
+	}
+	var terms []query.Query
+	var prefix []query.Query
+	for _, arg := range args {
+		terms = append(terms, bleve.NewTermQuery(arg))
+		prefix = append(prefix, bleve.NewPrefixQuery(arg))
+	}
+	queries = append(queries, bleve.NewConjunctionQuery(terms...))
+	queries = append(queries, bleve.NewConjunctionQuery(prefix...))
+
```

Comment:
```golang
var qs []query.Query
for _, arg := range args {
     qs = append(qs, bleve.NewTermQuery(arg), bleve.NewPrefixQuery(arg))
}
queries = append(queries, bleve.NewConjunctionQuery(qs...))
```

Reply:
The `terms` and `prefix` are individual and apply to different scenarios. e.g:
1. terms: `commit transaction` can match the content `commit the transaction`, we use `commit` and `transaction` to match the content first and conjunction them.
2. prefix: `commit trans` can match the content `commit the transaction`, we use `commit` and `trans` to match the content first and conjunction them.